config/api.yml

```diff
@@ -472,6 +472,12 @@
       :post:
       - :name: create
         :identifier: vm_miq_request_new
+    :resource_actions:
+      :post:
+      - :name: approve
+        :identifier: vm_miq_request_approve
+      - :name: deny
+        :identifier: vm_miq_request_deny
```

Comment:
also add the approve/deny actions to collection_actions so we can do the same on multiple provision requests in a single request.